test/unit/lib/plugins/aws/lib/naming.test.js

```diff
@@ -705,7 +705,7 @@ describe('#naming()', () => {
         sdk.naming.getLambdaCognitoUserPoolPermissionLogicalId(
           'functionName',
           'Pool1',
-          'CustomMessage'
+          'Custom-Message'
```

Comment:
Let's maybe use an actually valid `triggerSource` e.g. `PreSignUp_ExternalProvider` instead of a made-up one?

Reply:
Sure thing